flipperdevices/flipperzero-firmware
# [FL-2430] Automatic Desktop Locking

# What's new
- Automatic desktop locking
- Customizable timeout
- Backlight switches off quickly in locked mode

# Verification 
- Go to Settings -> Desktop -> Auto Lock Time and set it to anything other than OFF
- Exit all applications (that means Settings too)
- Wait for the timeout to occur - Flipper should go to the locked mode
- Press random buttons in locked mode - display backlight should turn off after 1 second regardless of the backlight timeout setting
- Unlock Flipper, go to any application.  Automatic locking should not occur when an applicaton is running.

# Checklist (For Reviewer)
